pandas/core/generic.py

```diff
@@ -1130,8 +1130,8 @@ def to_msgpack(self, path_or_buf=None, encoding='utf-8', **kwargs):
         return packers.to_msgpack(path_or_buf, self, encoding=encoding,
                                   **kwargs)
 
-    def to_sql(self, name, con, flavor='sqlite', schema=None, if_exists='fail',
-               index=True, index_label=None, chunksize=None, dtype=None):
+    def to_sql(self, name, con, schema=None, if_exists='fail', index=True,
+               index_label=None, chunksize=None, dtype=None, *args, **kwargs):
```

Comment:
I don't think the `*args` are needed here. If people were using positional kwargs, the current deprecation will not work anyways (as `schema` will catch the flavor)